.gitignore

```diff
@@ -1,6 +1,6 @@
 node_modules
 
-dist/
```

Comment:
Did you remove it to make the standalone available online ?

I would prefer to keep dist ignored

Reply:
Could we keep only the file that interest us (and maybe add versionin as well, dont know how). https://www.jsdelivr.com/ looks nice for hosting

Reply:
JSDelivery can use npm, Github repo and Wordpress git (https://www.jsdelivr.com/features), so I used Github feature to test in real condition (cdn compression) to know the size but it' can be definitely deleted.

If you push `dist` with NPM publish ([like today](https://www.jsdelivr.com/package/npm/@flagship.io/js-sdk?path=dist)), it's working :)

Reply:
We can make a versioning after package has been published with a custom script (github hook ?). The script could also upload it in the server

Reply:
Another option related to what did @Julien is to move the standalone version in another folder instead of `dist`, maybe `public` ?
So we keep dist folder ignored and expose only needed stuff